test/externalTests/ens.sh

```diff
@@ -28,22 +28,23 @@ verify_input "$@"
 BINARY_TYPE="$1"
 BINARY_PATH="$2"
 
-function compile_fn { npx truffle compile; }
-function test_fn { npm run test; }
+function compile_fn { yarn build; }
+function test_fn { yarn test; }
```

Comment:
I suspect that either would work but I thought it would be better to follow what the project does. In this case the original ENS repo had instructions using `npm` in the README and the new one tells you to use yarn (even has `yarn.lock`).